X10D/src/Math/ComparableExtensions.cs

```diff
@@ -133,7 +133,7 @@ public static T Clamp<T>(this T value, T lower, T upper)
         if (lower.GreaterThan(upper))
         {
             throw new ArgumentException(
-                string.Format(ExceptionMessages.LowerCannotBeGreaterThanUpper, lower, upper),
+                string.Format(null, ExceptionMessages.LowerCannotBeGreaterThanUpper, lower, upper),
```

Comment:
Can you modify this so that it defaults to `CultureInfo.CurrentCulture` rather than `null`?